### DISCUSSION TITLE
Druckmaßstab individuell einstellen - Wert übernehmen

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/webgis-community#116] [Q&A]
Hallo!

Wir haben Probleme beim individuellen Einstellen eines Druckmaßstabs. Wenn man den Maßstab direkt eingibt 
![image](https://github.com/gis-eni/webgis-forum/assets/127403099/4bf1fd88-46c5-4c42-89f9-7e4e9bf82699)

wird dieser nicht übernommen, sondern auf den nächstgelegen fix eingestellten Maßstab gerundet:

![image](https://github.com/gis-eni/webgis-forum/assets/127403099/4acc6527-84e4-4622-9e53-5de30540c123)

Wo kann man das einstellen/korrigieren? Der Wert blinkt kurz auf und wird dann übersteuert.

Muss man viell. die layout scales im Drucklayout weglassen?
![image](https://github.com/gis-eni/webgis-forum/assets/127403099/f1209a22-7b85-4ccb-a5d5-b5fb2957fb10)

**Comment by gis-eni** [comment]
Hallo!

Wenn du im Drucklayout Maßstabe einstellst, kann können explizit nur diese mit dem Layout ausgedruckt werden. Das ist per Definition so. Die Möglichkeit mit der Angabe von Maßstäben im Layoutfile wurde extra dafür eingeführt: Der Anwender kann nur in bestimmten Maßstäben ausdrucken. Das zB bei Einreichplänen Sinn, die nur im Maßstab 1:10000 oder 1:5000. Das du schon im Layout alle Maßstäbe drinnen hast, zeigt, dass ihr was falsch konfiguriert wurde.

Normalerweise gibt man die Druckmaßstäbe in der api.config an. Der Anwender kann dann auch eigene eingeben:

https://docs.webgiscloud.com/cloud/selfhosting/install/config/api/index.html#werkzeug-drucken

**Comment by ue-black** [comment]
Wir haben die Maßstäbe in den Drucklayout xmls auch angegeben, dadurch war eine individuelle Eingabe nicht mehr möglich. Nach Löschen aus dem xml hat es jetzt gepasst. Danke!

**Comment by gis-eni** [comment]
Verbesserung: Im Dialog Ok Button einbauen

**Reply by gis-eni** [reply]
Ok Button wurde hinzugefügt:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/f2676422-9be9-435a-a7d2-73b0f8af690e)
